### ISSUE TITLE
Workspaces disappear from interface after a stop/crash 

### ISSUE THREAD

**Issue by cifren** [eclipse-che/che#3352]
First thank you for all that awesome work you do !!

The problem : Anytime the system doesn't close correctly, my workspaces disappear from interface. 

I use windows with Boot2Docker. And I use [Docker Launcher](https://eclipse-che.readme.io/docs/usage-docker).

When the VM shutdown correctly, respecting the rules, everything is fine. But when the VM crash or is terminated brutally, the workspaces are not there anymore... Everything is reset. 

Anytime the system crash, I need to run <docker rm -f che-server>, which seems normal because the server didn't exit normally.

When I access to my VM, it seems that the folder "/home/user/che/workspace" is not bind/mount anymore. But in docker, the workspaces are still there.

Is there a command to get them back ?

**Reproduction Steps:**

1. Start che-server and prepare a workspace
2. Stop the OS without any command (just close)
3. <docker rm -f che-server> and start che-server

**Expected behavior:**

Workspace still there

**Observed behavior:**

No more workspace

**Che version:**    5.0.0-latest
**OS and version:** Win7 + Boot2Docker
**Docker version:**  Version:      1.12.3  
**Che install:**    Docker container

* Problem started happening recently, didn't happen in an older version of Che:  I Don't know
* Problem can be reliably reproduced, doesn't happen randomly:  No

**Comment by TylerJewell** [comment]
If you are using boot2docker on windows 7, the only host files that are bind mounted are those in %userprofile%.  So this may be an issue with boot2docker and no so much Che.  Can you elaborate upon where you are telling Che to mount between boot2docker and the inner VM?

**Comment by cifren** [comment]
I am new with docker, so I am still exploring the world there.

I just finished some tests, and when you use [volumes](https://docs.docker.com/engine/tutorials/dockervolumes/) if the VM crashes, boot2docker will destroy the folder somehow and we loose all data that was in it. 

For example: 
``` shell
docker run -it -v /root/tchat:/web ubuntu
```
"/root/tchat" is on VM boot2docker and "/web" on the docker Ubuntu, if it crashes, "/root/tchat" disappears... not sure why... maybe because it is mounted folder or something like this.

So I managed to make it better and persistent.

``` shell
docker run --rm -t -e CHE_DATA=/c/Users/webide -v /var/run/docker.sock:/var/run/docker.sock eclipse/che:5.0.0-latest start
```

The doc has some line written about it, but it doesn't say that is not persistent so I didn't think at that moment it is was important, maybe an explicit warning for the windows7 users on the doc would be useful ? Something under "Usage" or "Start" section ?

Thank you for your help, %userprofile% gave me a hint that made my day.

Next step, Rule the world (of Eclipse Che at least)

:)
